src/data/redux/thunks.ts

```diff
@@ -0,0 +1,27 @@
+import { Dispatch } from "redux";
+import { remoteState } from "../../core/backendStateSync";
+import { logError } from "../../helpers/core/customLogger";
+import { ActionModel } from "../state/actionModel";
+import { ActionType } from "../state/actionType";
+import { StateModel } from "../state/stateModel";
+
+
+export const initStateThunk = ()=>{
+    return (dispatch:Dispatch)=>{
+        return remoteState.getAsync()
+        .then((state:StateModel)=>{
+            dispatch({...new ActionModel(ActionType.SET_EDITOR, state.editorState)});
```

Comment:
`nitpick`

Passing some other than plain object structures to `dispatch` may cause issues with serialization ([some thought on that in redux docs](https://redux.js.org/faq/design-decisions#why-doesnt-redux-support-using-classes-for-actions-and-reducers)).